repository/src/main/java/org/apache/atlas/GraphTransactionInterceptor.java

```diff
@@ -167,6 +171,22 @@ public Object invoke(MethodInvocation invocation) throws Throwable {
         }
     }
 
+    public boolean checkForBatchTooLargeError(Throwable t) {
+        Throwable currentCause = t;
+        while (currentCause != null) {
+            String message = currentCause.getMessage();
+            if (message != null &&
+                    message.contains("Batch too large") &&
```

Comment:
String.contains is case sensitive, I hope message contains exactly same case

Reply:
Yep its the same everytime